keyserver/listener.go

```diff
@@ -55,7 +55,8 @@ func (server *ConiksServer) acceptClient(conn net.Conn, handler func(msg []byte)
 
 	var buf bytes.Buffer
 	if _, err := io.CopyN(&buf, conn, 8192); err != nil && err != io.EOF {
-		log.Printf("client read %v: %v", conn.RemoteAddr(), err)
+		server.logger.Error(err.Error(),
+			zap.String("address", conn.RemoteAddr().String()))
```

Comment:
Is there a way we can completely wrap `zap` in our logger wrapper? It would be nice to just be able to write something like `server.logger.Error(err.Error(), "address", conn.RemoteAddr().String())`